ISISComputingGroup/ibex_gui
# Ticket2441: Pressure graphs should be logarithmic

### Description of work

Changed all pressure graphs on OPIs to logarithmic. As per https://github.com/ISISComputingGroup/IBEX/issues/2441

### To test

Confirm graphs on the following OPIs have logarithmic scales:
* PDR2000
* TPG (300/26x)
* amint2l

Confirm that there are no other pressure gauges to change

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
